backend/src/main/java/gov/cdc/usds/simplereport/db/repository/PhoneNumberRepository.java

```diff
@@ -14,4 +15,10 @@ public interface PhoneNumberRepository extends AuditedEntityRepository<PhoneNumb
   List<List<PhoneNumber>> findAllByPersonInternalIdIn(Collection<UUID> personIds);
 
   List<PhoneNumber> findAllByInternalIdIn(Collection<UUID> phoneIds);
+
+  @Query(
+      value =
+          "select * from {h-schema}phone_number where internal_id in (select p.primary_phone_internal_id from {h-schema}person p where p.internal_id in :personIds)",
+      nativeQuery = true)
+  List<PhoneNumber> findPrimaryPhoneNumberByPersonInternalIdIn(Collection<UUID> personIds);
```

Comment:
Again sub-queries tended to perform the best when running with `EXPLAIN` and it honestly easier to read.